test/Fs.re

```diff
@@ -77,10 +77,10 @@ let%test "rename - can rename a directory" = {
     let f = (srcTempPath, dstTempPath) => {
       open RunAsync.Syntax;
       let src = Path.(srcTempPath / "test.txt");
-      let dst = Path.(dstTempPath / "");
+      let dst = dstTempPath;
```

Comment:
Why this change happened? I believe it may change the path from `/folder/` to `/folder`

Reply:
`Path.(dstTempPath / "")` gives an error because
Internally it calls `Fpath.of_string` with `""` which results in error
https://erratique.ch/software/fpath/doc/Fpath/index.html#val-of_string

Reply:
Oh I already hit that in the past, makes sense